src/components/base/BaseSelect.vue

```diff
@@ -27,7 +27,7 @@
 import Multiselect from "vue-multiselect";
 
 export default {
-  name: "MovieSelect",
+  name: "baseSelect",
```

Comment:
Good to see that you introduce naming convention for base components. 
I think there might be typo since it's 'baseSelect' now and probably should be written with capital 'B'. ^_^